source/_integrations/lutron_caseta.markdown

```diff
@@ -174,6 +176,14 @@ Because Lutron Caseta devices automatically report state to Home Assistant (rath
 
 For more information on working with binary sensors in Home Assistant, see the [Binary Sensors Component](/components/binary_sensor/)
 
+## Button
+
+After setup, buttons will appear in Home Assistant using an `entity_id` based on the name used in the Lutron mobile app. For example, a button 'Button 1' on a keypad called 'Front Entrance Keypad' will appear in Home Assistant as `button.front_entrance_keypad_button_1`.
+
+For more information on working with buttons in Home Assistant, see the [Buttons component](/integrations/button/).
+
+Available services: `button.press`
+
```

Comment:
```suggestion
```

We probably don't need to document services provided by the underlying entity class since they are documented on the `button` integration

Reply:
I'm just matching the platform documentation in the rest of the document.

```
## Switch

After setup, switches will appear in Home Assistant using an `entity_id` based on the name used in the Lutron mobile app. For example, a light switch called 'Master Bathroom Vanity' will appear in Home Assistant as `switch.master_bathroom_vanity`.

For dimmable lights including wall and plug-in dimmers, see the light section on this page.

For more information on working with switches in Home Assistant, see the [Switches component](/integrations/switch/).

Available services: `switch.turn_on` and `switch.turn_off`.
```

Reply:
We can remove it from all the platforms in this doc to be consistent?

Reply:
> We can remove it from all the platforms in this doc to be consistent?

Let's do that

Reply:
ok, done.